orchagent/aclorch.h

```diff
@@ -391,6 +425,11 @@ class AclOrch : public Orch, public Observer
     sai_object_id_t getTableById(string table_id);
     const AclTable* getTableByOid(sai_object_id_t oid) const;
 
+    static mutex& getCountersMutex()
```

Comment:
Actually this mutex is no longer needed. Few releases back orchagent spawned a thread for counter polling and this mutex was introduced. Later approach with SelectableTimer was used. So, orchagent is fully single threaded now.

Reply:
@stepanblyschak Thanks! Legacy code is always confusing...